Polish the doc of v1.0.3 release (#190)
harvester/docs: docs/troubleshooting/monitoring.md

## Monitoring is unusable

When the Harvester Dashboard is not showing any monitoring metrics, it can be caused by the following reasons.

> ### Monitoring is unusable due to Pod stuck in `Terminating`

Reviewer: ```suggestion
### Monitoring is unusable due to Pod being stuck in `Terminating` status
```

> Harvester Monitoring related pods are deployed randomly in the cluster Nodes, when the Node hosting following pods is accidentally down, the related pods may stuck in status `Terminating`, then the Monitoring is unusable from the WebUI.

Reviewer: ```suggestion
Harvester Monitoring pods are deployed randomly on the cluster Nodes. When the Node hosting the pods accidentally goes down, the related pods may become stuck in the `Terminating` status rendering the Monitoring unusable from the WebUI.
```

```

> Monitoring can be recovered via using CLI commands to force delete the related pods, and the cluster will redeploy new pods to replace them.

Reviewer: ```suggestion
Monitoring can be recovered using CLI commands to force delete the related pods. The cluster will redeploy new pods to replace them.
```

$ kubectl delete pod --force -n cattle-monitoring-system rancher-monitoring-grafana-d9c56d79b-t24sz
```

> Wait a few minutes, the new pods are created and become ready. The Monitoring dashboard will be usable again.

Reviewer: ```suggestion
Wait for a few minutes so that the new pods are created and readied for the Monitoring dashboard to be usable again.
```
